packages/common/src/ui/icons/User.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon';
+
+export const UserIcon = (props: SvgIconProps): JSX.Element => {
```

Comment:
and here is where I regret calling icons 'Something' and not 'SomethingIcon' 🙄

Reply:
haha yes. I think it's not too late and probably a good idea